src/augeas.c

```diff
@@ -2020,6 +2021,39 @@ int aug_escape_name(augeas *aug, const char *in, char **out) {
     return result;
 }
 
+int aug_filter_file(struct augeas *aug, const char *file) {
```

Comment:
Since this now actually loads the file, I think it should be renamed to `aug_load_file` (and similar for the augtool command and command line option)